avro-fastserde/src/main/java/com/linkedin/avro/fastserde/backport/Symbol.java

```diff
@@ -481,10 +481,20 @@ public static EnumAdjustAction enumAdjustAction(int rsymCount, Object[] adj) {
   }
 
   public static class EnumAdjustAction extends IntCheckAction {
+    public final boolean noAdjustments;
     public final Object[] adjustments;
+
     @Deprecated public EnumAdjustAction(int rsymCount, Object[] adjustments) {
       super(rsymCount);
       this.adjustments = adjustments;
+      boolean noAdj = true;
+      if (adjustments != null) {
+        int count = Math.min(rsymCount, adjustments.length);
+        noAdj = (adjustments.length <= rsymCount);
```

Comment:
Why not use `>=` here to make it more readable?

Reply:
Also can you leave some comments on these new logics?

Reply:
Same here, I just copied this from avro Symbol class